pkg/initializer/initializer.go

```diff
@@ -54,11 +54,9 @@ func (e *EtcdInitializer) Initialize(mode validator.Mode, failBelowRevision int6
 	//Etcd cluster scale-up case
 	if miscellaneous.IsMultiNode(e.Logger.WithField("actor", "initializer")) {
 		m := member.NewMemberControl(e.Config.EtcdConnectionConfig)
-		isEtcdMemberPresent, err = m.IsMemberInCluster(ctx)
-		if !isEtcdMemberPresent && err == nil {
-			retry.OnError(retry.DefaultBackoff, func(err error) bool {
-				return err != nil
-			}, func() error {
+		isScaleup, err := member.IsScaleup(ctx, m)
+		if isScaleup && err == nil {
+			retry.OnError(retry.DefaultBackoff, errors.AnyError, func() error {
```

Comment:
if one is sure that a cluster has been scaled up from nonHA to HA then there can be 2 approaches:
1. Infinitely attempt to add a member as learner till it succeeds
2. Try for fixed number of times and then exit causing a restart of backup-restore.
One should not proceed ahead without adding this member as a learner.